README.md

```diff
@@ -92,17 +92,24 @@ Notable features:
 Pricing [here](https://www.warpbuild.com/pricing).
 
 Notable features:
-- 1-line change to get faster and cheaper builds in most projects.[^warp-1click]
-- Support for x86-64 and ARM runners with unlimited concurrency.
+- 1-line change to get faster and cheaper builds in most projects[^warp-1click].
+- Support for x86-64 and ARM runners with free unlimited concurrency.
+- 150GB disk storage with unlimited fast caches and `setup-*` actions[^warp-cache].
+- Bring Your Own Cloud to run workflows in your AWS account and region[^warp-byoc].
+- Customizable runner, disk, and processor configurations[^warp-customrunners].
 - **Live debug failed jobs over SSH.**[^warp-debug]
 - Discounts for open-source projects and young startups, and referral programs to get extra minutes.[^warp-pricing]
 - Support for macOS runners on M2 Pros.[^warp-runners]
-- **(Coming soon)** Automated container layer caching and AI powered analytics (~Q1 2024). 🚧
+- Container layer caching, SOC2 Type 2, and 10x faster incremental builds (~Aug 2024). 🚧
```

Comment:
Hey, is still WIP? If so, the date seems wrong. Did you mean August 2025 by any chance?

Reply:
Oof, it's probably because the PR is quite old. Sorry about that. Would you mind updating, and then I'll merge? @suryaoruganti thanks :)